### PR TITLE
Add instructions on how to use FINAL modifier

### PR DESCRIPTION
**Pull request by hkrutzer**
Not sure this is the best location, I can put it somewhere else.

### CODE REVIEW

**Comment by ruslandoga** [comment]
👋 @hkrutzer 

What's special about `FINAL` that we need to document it separately?

**Comment by hkrutzer** [comment]
After using Ecto for several years, I've never had to put a keyword after the table name, so I spent around 20 minutes looking for how to do it. I also didn't immediately realize it can be passed as a setting. I suspect `FINAL` is used relatively often with Clickhouse so I thought I'd save the next person some time.

I could also only add it to the settings examples and someone ctrl+f'ing FINAL could still find it, and remove the `hints` version.

**Comment by ruslandoga** [comment]
> I could also only add it to the settings examples 

I like this approach better. Ability to pass [hints](https://hexdocs.pm/ecto/Ecto.Query.html#join/5-hints) in queries seems more of Ecto concern.

**Comment by hkrutzer** [comment]
Updated accordingly. Should save the next person looking for `FINAL` a few minutes.

File: README.md [hunk 1]
```diff
@@ -87,11 +87,12 @@ MyApp.Repo.insert_all("example", rows, types: types)
 
 #### Settings
 
-`:settings` option can be used to enable [asynchronous inserts,](https://clickhouse.com/docs/en/optimize/asynchronous-inserts) lightweght [deletes,](https://clickhouse.com/docs/en/guides/developer/lightweght-delete) and [more:](https://clickhouse.com/docs/en/operations/settings/settings)
+`:settings` option can be used to enable [asynchronous inserts,](https://clickhouse.com/docs/en/optimize/asynchronous-inserts) lightweight [deletes,](https://clickhouse.com/docs/en/guides/developer/lightweght-delete) [FINAL modifier,](https://clickhouse.com/docs/en/sql-reference/statements/select/from#final-modifier) and [more:](https://clickhouse.com/docs/en/operations/settings/settings)
 
 ```elixir
 MyApp.Repo.insert_all(MyApp.Example, rows, settings: [async_insert: 1])
 MyApp.Repo.delete_all("example", settings: [allow_experimental_lightweight_delete: 1])
+MyApp.Repo.insert_all(MyApp.Example, rows, settings: [final: 1])
```
Comment (ruslandoga) [inline README.md:95]: You probably meant `MyApp.Repo.all`

Comment (hkrutzer) [reply README.md]: Oops, sorry

File: README.md [hunk 2]
```diff
@@ -87,11 +87,12 @@ MyApp.Repo.insert_all("example", rows, types: types)
 
 #### Settings
 
-`:settings` option can be used to enable [asynchronous inserts,](https://clickhouse.com/docs/en/optimize/asynchronous-inserts) lightweght [deletes,](https://clickhouse.com/docs/en/guides/developer/lightweght-delete) and [more:](https://clickhouse.com/docs/en/operations/settings/settings)
+`:settings` option can be used to enable [asynchronous inserts,](https://clickhouse.com/docs/en/optimize/asynchronous-inserts) lightweight [deletes,](https://clickhouse.com/docs/en/guides/developer/lightweght-delete) [FINAL modifier,](https://clickhouse.com/docs/en/sql-reference/statements/select/from#final-modifier) and [more:](https://clickhouse.com/docs/en/operations/settings/settings)
 
 ```elixir
 MyApp.Repo.insert_all(MyApp.Example, rows, settings: [async_insert: 1])
 MyApp.Repo.delete_all("example", settings: [allow_experimental_lightweight_delete: 1])
+MyApp.Repo.all(MyApp.Example, rows, settings: [final: 1])
```
Comment (ruslandoga) [inline README.md:95]: Maybe something like

```elixir
MyApp.Repo.all(MyApp.Example, settings: [final: 1])
```

**Comment by ruslandoga** [comment]
@hkrutzer thank you!

**Pull request merged**